shared/output/cloud/compute_small/lib/google/cloud/compute/v1/compute_small_services_pb.rb

```diff
@@ -0,0 +1,76 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Can we not generate this file if we don't have a gRPC client?

Reply:
yes, I am going to update the Rake file with this